cell_based/src/cell/srn/DeltaNotchEdgeSrnModel.hpp

```diff
@@ -172,40 +183,42 @@ class DeltaNotchEdgeSrnModel : public AbstractOdeSrnModel
      *
      * @param rParamsFile the file stream to which the parameters are output
      */
-    virtual void OutputSrnModelParameters(out_stream& rParamsFile) override;
+    virtual void OutputSrnModelParameters(out_stream& rParamsFile);
 
     /**
-     * Adds Delta/Notch from the input srn model to this model.
+     * Adds Delta/Notch from the input SRN model to this model.
      * Override the method declared in AbstractSrnModel class
-     * @param p_other_srn
-     * @param scale
+     * 
+     * @param pOtherSrn Pointer to an SRN
+     * @param scale scale factor
      */
-    virtual void AddSrnQuantities(AbstractSrnModel *p_other_srn,
-                                  const double scale = 1.0) override;
+    virtual void AddSrnQuantities(AbstractSrnModel* pOtherSrn,
+                                  const double scale = 1.0);
 
     /**
      * Here we assume that when a neighbouring junctions shrinks, 25% of its Delta/Notch
      * concentration is added to this edge
      * Override the method declared in AbstractSrnModel class
-     * @param p_shrunk_edge_srn
+     * 
+     * @param pShrunkEdgeSrn Pointer to an SRN
      */
-    virtual void AddShrunkEdgeSrn(AbstractSrnModel *p_shrunk_edge_srn) override;
-
-    // adding comment for testing only, to be removed.
+    virtual void AddShrunkEdgeSrn(AbstractSrnModel* pShrunkEdgeSrn);
 
     /**
      * Here we add Delta/Notch when junctions merge via common vertex deletion
      * Override the method declared in AbstractSrnModel class
-     * @param p_merged_edge_srn
+     * 
+     * @param pMergedEdgeSrn Pointer to an SRN
      */
-    virtual void AddMergedEdgeSrn(AbstractSrnModel* p_merged_edge_srn) override;
+    virtual void AddMergedEdgeSrn(AbstractSrnModel* pMergedEdgeSrn);
 
     /**
      * By default, Edge concentrations are split according to relative lengths, when an edge is split.
      * Override the method declared in AbstractSrnModel class
-     * @param relative_position
+     * 
+     * @param relativePosition relative position to use when splitting Edge concentrations
```

Comment:
Can we explain what this `relativePosition` means a bit more? This is a bit of a tautology comment!

Reply:
This makes sense as it is already explained a little bit more in the AbstractSrnModel which this class inherits from. Abstract has it define this way in the comment: 

"Scales SRN quantities due to edge split. Amount of scaling may depend on the resulting proportions, determined by relative position of the node that splits the edge."

So something along the lines of "Relative position of the new node which splits the two edges" or something along those lines could be added here. Do you think that is sufficient ?

Reply:
Yeah, it wants to say relative position of what relative to what.

Reply:
Changes required by @mirams were made by @BJackal in commits abd66f0, f51bda9, and 88730a2.